util/reconciler/defaults.go

```diff
@@ -25,10 +29,60 @@ const (
 	DefaultMappingTimeout = 60 * time.Second
 )
 
+var AllowedErrors = []int{429, 503}
```

Comment:
Change to ThrottlingErrors or add a comment

Reply:
Ok done